dataland-frontend/src/components/generics/ViewFrameworkBase.vue

```diff
@@ -170,22 +171,37 @@ export default defineComponent({
      * @param event event
      */
     editDataset(event: Event) {
+        console.log(0);
+        console.log({
+            mapOfReportingPeriodToActiveDataset: this.mapOfReportingPeriodToActiveDataset,
+            singleDataMetaInfoToDisplay: this.singleDataMetaInfoToDisplay
+        });
```

Comment:
please delete remaining debug console.logs

Reply:
Already deleted. This was a "work in progress" commit.

Reply:
Ah ok sorry I did not know that 👍